osf/migrations/0212_registrationschema_providers.py

```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
```

Comment:
Could we go ahead and make this 0213, so we don't have multiple 0212 migrations?